packages/handlers/graphql/src/index.ts

```diff
@@ -1,10 +1,15 @@
 import { GetMeshSourceOptions, MeshHandler, MeshSource, ResolverData, YamlConfig } from '@graphql-mesh/types';
 import { fetchache, KeyValueCache, Request } from 'fetchache';
+import { CodeFileLoader } from '@graphql-tools/code-file-loader';
+import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader';
```

Comment:
More loaders are available but these two were chosen as they are already used by `generate-sdk`. For example, GraphQL Code Generator [uses 8 loaders](https://github.com/dotansimha/graphql-code-generator/blob/cbd9970/packages/graphql-codegen-cli/src/load.ts#L23-L32).